website/photos/templates/photos/fotopondwidget.html

```diff
@@ -0,0 +1,51 @@
+{% load static %}
+<style>
+    .filepond--panel-root {
+        background-color: var(--darkened-bg);
+    }
+
+    .filepond--credits {
+        display: none;
+    }
+    .filepond {
+        min-width: 200px;
+        min-height: 76px;
+    }
+</style>
+
+<input type="file" class="filepond">
+
+<!-- Load FilePond library -->
+<script type="text/javascript" src="{% static "photos/js/filepond.min.js" %}"></script>
+<script type="text/javascript" src="{% static "js/js.cookie.min.js" %}"></script>
+
+<!-- Turn all file input elements into ponds -->
+<script>
+    FilePond.parse(document.body);
+    FilePond.setOptions({
+        name: "{{ widget.name }}",
+        chunkUploads: true,
+        chunkSize: 500000,
+        dropOnPage: true,
+        allowMultiple: false,
+        allowPaste: true,
```

Comment:
It would be really nice to set these options on the `FilePondWidget` for reusability